### DISCUSSION TITLE
Support for VPN Unlimited

### DISCUSSION THREAD

**Discussion by sunniguy** [qdm12/gluetun#488] [Ideas]
Hello all. I was wondering if support for VPN Unlimited could be added? 
https://www.vpnunlimited.com/help/manuals/how-to-manually-create-vpn-conf

Thank you.

**Comment by qdm12** [comment]
Please subscribe to the issue https://github.com/qdm12/gluetun/issues/420 it's a work in progress, almost done.

Actually if you want to help, I have a quick question: do you use username and password for connecting to openvpn servers, or just the RSA private key? Thanks!

**Reply by sunniguy** [reply]
More than happy to help, however not entirely sure on that. You generate a manual configuration on login which gives you a QR code, a server address, a global config login, and a device config login. It also gives you a .ovpn file which contains certificate information. I can't tell which login info the certificate contains.

**Reply by qdm12** [reply]
So no username and password right? What do you mean by global Config login?

Someone sent me a dump of openvpn files. Certificates are not for user authentication, but there is a unique rsa key by openvpn file which might be for authentication. I guess I can try, I was just being lazy on That, I'll get back to you.

**Reply by sunniguy** [reply]
No login information as such. The .ovpn file does contain a "private key" - could that be used for authentication?

"global config login" - the username is the same as the login email address, and the password is a new one generated when you make the ovpn file. Looks like as last screenshot here: https://www.vpnunlimited.com/help/manuals/how-to-manually-create-vpn-conf

All the login information is the same for all the different server .ovpn files.
